configs/next/packages/gcc/gcc.mk

```diff
@@ -59,13 +53,8 @@ $(RCPTS)/gcc_1.a.rcpt: $(RCPTS)/binutils_1.rcpt \
                        $(RCPTS)/rsync_gcc.rcpt
 	@touch $@
 
-$(RCPTS)/gcc_2.a.rcpt: $(RCPTS)/gcc_1.rcpt \
-                       $(RCPTS)/glibc_1.rcpt \
-                       $(RCPTS)/ldconfig_1.rcpt
-	@touch $@
-
 $(RCPTS)/gcc_3.a.rcpt: $(RCPTS)/ldconfig_1.rcpt \
-                       $(RCPTS)/gcc_2.rcpt
+                       $(RCPTS)/glibc_1.rcpt
```

Comment:
This removes some dependencies (`gcc_1`, `ldconfig_1`). Is that intentional?

Reply:
@ThinkOpenly note the `ldconfig_1` dependency was already part of gcc_3 (the white line not in the diff). But I have the same question regarding the `gcc_1` dep.

Reply:
@tuliom Makes sense. The `gcc_1` dep is satisfied implicitly through `glibc_1`, so removing that  really doesn't change anything in the end. Thanks for the explanation!

Reply:
OK